x-pack/plugins/cloud_security_posture/public/pages/findings/findings_container.tsx

```diff
@@ -38,54 +40,57 @@ export const getDefaultQuery = (): CspFindingsRequest & { groupBy: GroupBy } =>
   groupBy: 'none',
 });
 
-const getGroupByOptions = (): Array<EuiComboBoxOptionOption<GroupBy>> => [
+const getGroupByOptions = (): Array<EuiComboBoxOptionOption<FindingsBaseURLQuery['groupBy']>> => [
   {
     value: 'none',
     label: i18n.translate('xpack.csp.findings.groupBySelector.groupByNoneLabel', {
       defaultMessage: 'None',
     }),
   },
   {
-    value: 'resourceType',
+    value: 'resource',
     label: i18n.translate('xpack.csp.findings.groupBySelector.groupByResourceTypeLabel', {
       defaultMessage: 'Resource Type',
     }),
```

Comment:
@orouz since we're grouping by resource id, can we change this to `Resource ID`?

Reply:
Hey, and the table column should be `Resource` as well? or `Resource Id`?

Reply:
The table column should be `Resource Id`